packages/next/src/build/utils.ts

```diff
@@ -640,19 +640,35 @@ export async function printTreeView(
         .map((e) => e.replace(buildId, '<buildId>'))
         .sort(),
       ...sharedCssFiles.map((e) => e.replace(buildId, '<buildId>')).sort(),
-    ].forEach((fileName, index, { length }) => {
-      const innerSymbol = index === length - 1 ? '└' : '├'
+    ]
+
+    // if the some chunk are less than 10kb or we don't know the size, we only show the total size of the rest
+    const tenKbLimit = 10 * 1000
+    let restChunkSize = 0
+    let restChunkCount = 0
+    sharedJsChunks.forEach((fileName, index, { length }) => {
+      const innerSymbol = index + restChunkCount === length - 1 ? '└' : '├'
 
       const originalName = fileName.replace('<buildId>', buildId)
       const cleanName = getCleanName(fileName)
       const size = stats.sizes.get(originalName)
 
+      if (!size || size < tenKbLimit) {
+        restChunkCount++
+        restChunkSize += size || 0
+        return
+      }
+
+      messages.push([`  ${innerSymbol} ${cleanName}`, prettyBytes(size), ''])
+    })
+
+    if (restChunkCount > 0) {
       messages.push([
-        `  ${innerSymbol} ${cleanName}`,
-        typeof size === 'number' ? prettyBytes(size) : '',
+        `  └ other shared chunks (total)`,
```

Comment:
```suggestion
        `  └ other shared chunks (total ${restChunkCount} chunks)`,
```
Should we log the chunk count for visibility?

Reply:
We can skip this IMO as they're pretty small ones, the purpose was to show less if the size are not necessarily concerned